apps/spruce/src/pages/image/index.tsx

```diff
@@ -12,34 +17,45 @@ const Image: React.FC = () => {
     [slugs.tab]: ImageTabRoutes;
   }>();
 
-  if (!Object.values(ImageTabRoutes).includes(currentTab)) {
+  if (!Object.values(ImageTabRoutes).includes(currentTab) && imageId) {
     return (
       <Navigate
         replace
-        // @ts-expect-error: TODO fix in DEVPROD-7654
         to={getImageRoute(imageId, ImageTabRoutes.BuildInformation)}
       />
     );
   }
 
-  return (
-    <SideNav aria-label="Image" widthOverride={250}>
-      <SideNavGroup>
-        {Object.values(ImageTabRoutes).map((tab) => (
-          <SideNavItem
-            active={tab === currentTab}
-            as={Link}
-            key={tab}
-            // @ts-expect-error: TODO fix in DEVPROD-7654
-            to={getImageRoute(imageId, tab)}
-            data-cy={`navitem-${tab}`}
-          >
-            {getTabTitle(tab).title}
-          </SideNavItem>
-        ))}
-      </SideNavGroup>
-    </SideNav>
-  );
+  if (imageId) {
```

Comment:
perhaps if `imageId` is not defined we should default to using the first image in the images list

Reply:
you might be interested in looking at one of the hooks we have `useFirstDistro.ts` as a reference

Reply:
resolved